fml/raster_thread_merger.cc

```diff
@@ -17,23 +17,37 @@ RasterThreadMerger::RasterThreadMerger(fml::TaskQueueId platform_queue_id,
       gpu_queue_id_(gpu_queue_id),
       task_queues_(fml::MessageLoopTaskQueues::GetInstance()),
       lease_term_(kLeaseNotSet) {
-  is_merged_ = task_queues_->Owns(platform_queue_id_, gpu_queue_id_);
+  bool is_merged = task_queues_->Owns(platform_queue_id_, gpu_queue_id_);
+  is_merged_.store(is_merged);
+  if (is_merged) {
+    lease_term_ = 1;
+  }
 }
 
 void RasterThreadMerger::MergeWithLease(size_t lease_term) {
   FML_DCHECK(lease_term > 0) << "lease_term should be positive.";
-  if (!is_merged_) {
-    is_merged_ = task_queues_->Merge(platform_queue_id_, gpu_queue_id_);
+  if (!is_merged_.load()) {
+    FML_DLOG(ERROR) << "--- tm_: merged";
```

Comment:
is this intended?

Reply:
No, I left the comments here for testing purpose as I still need to add tests. Will when landing this.